src/modules/admin/admin.controller.ts

```diff
@@ -13,17 +13,22 @@ import {
 import { RolesGuard } from './guard/roles.guard';
 import { UsersService } from '../users/users.service';
 import {
+  GetAdminProductsQueryDto,
   SortableField,
   SortableOption,
   SortableRole,
-} from './dto/sort-option.dto';
-import { UpdateStatusDto } from './dto/update-status.dto';
-import { User } from '~/entities';
+  UpdateStatusDto,
+} from './dto/index';
+import { ProductEntity, User } from '~/entities';
+import { ProductsService } from '../products/products.service';
 
 @Controller('admin')
 @UseGuards(RolesGuard)
 export class AdminController {
-  constructor(private readonly usersService: UsersService) {}
+  constructor(
+    private readonly usersService: UsersService,
+    private readonly productsService: ProductsService,
+  ) {}
 
   @Get('user/:id')
```

Comment:
@ApiOperation({
    summary: '...',
    description: '...',

Reply:
Added api tag for the controller